x-pack/plugins/observability_solution/investigate/public/index.ts

```diff
@@ -20,15 +20,17 @@ import type {
 export type { InvestigatePublicSetup, InvestigatePublicStart, OnWidgetAdd, WidgetRenderAPI };
 
 export {
-  type InvestigateTimeline,
+  type Investigation,
```

Comment:
I am wondering why we export these types from both public and common folders? I was looking into the consumer side where it is imported from and in some places we import it from common `'@kbn/investigate-plugin/common';`, where as in others from public `'@kbn/investigate-plugin/public';`

nit: Maybe we can keep only one place where we export/import from, unless there is a restriction I am not aware of

Reply:
this is for other plugins to import it (without extra config, other plugins cannot import from `/common`). the difference that you see might be based on type imports vs code imports.